readthedocs/builds/syncers.py

```diff
@@ -140,7 +140,8 @@ def copy(cls, path, target, host, is_file=False, **__):
         )
         ret = os.system(sync_cmd)
         if ret != 0:
-            log.info("COPY ERROR to app servers.")
+            log.error("COPY ERROR to app servers. Return code: {}".format(ret))
+            log.error("COPY ERROR command run: {}".format(sync_cmd))
```

Comment:
One log line would be better here, so we're not splitting reporting up into two line/pieces to sentry.